rust/rope/src/spans.rs

```diff
@@ -155,6 +155,12 @@ pub struct SpanIter<'a, T: 'a + Clone + Default> {
     ix: usize,
 }
 
+pub struct SpanBackwardsIter<'a, T: 'a + Clone + Default> {
+    cursor: Cursor<'a, SpansInfo<T>>,
+    ix: usize,
+    done: bool,
+}
+
```

Comment:
I've implemented `DoubleEndedIterator` instead in https://github.com/google/xi-editor/pull/286/commits/3588ad811650efad64f7a2d627aed08dc01ea26c
I like doing the it the Rust way, if there is one (and I wasn't really aware of `DoubleEndedIterator` previously).